### PR TITLE
Add ResourceDistribution Doc

### PR DESCRIPTION
**Pull request by veophi**
Signed-off-by: veophi <vec.g.sun@gmail.com>

Add ResourceDistribution Doc.

### CODE REVIEW

File: docs/user-manuals/resourcedistribution.md [hunk 1]
```diff
@@ -0,0 +1,223 @@
+---
+title: ResourceDistribution
+---
+
+For the scenario, where the namespace-scoped resources such as Secret and ConfigMap need to be distributed or synchronized to different namespaces, the native k8s currently only supports manual distribution and synchronization by users one-by-one, which is very inconvenient. 
+
+Typical example: when users need to use the imagePullSecrets capability of SidecarSet, they must repeatedly create corresponding Secrets in relevant namespaces, and ensure the correctness and consistency of these Secret configurations.
```
Comment (furykerry) [inline docs/user-manuals/resourcedistribution.md:7]: can we give a non-sidecarset example? it is common that one need to share some environment variables across namespace

File: docs/user-manuals/resourcedistribution.md [hunk 2]
```diff
@@ -0,0 +1,223 @@
...
+
+Therefore, in the face of these scenarios that require the resource distribution and synchronization **across namespaces**, we provide a convenient tool, namely **ResourceDistribution**, to do this automatically.
```
Comment (furykerry) [inline docs/user-manuals/resourcedistribution.md:9]: plz emphasize that we **continuously synchronization** configmap&secrets

File: docs/user-manuals/resourcedistribution.md (hunk 1 above)
Comment (veophi) [reply docs/user-manuals/resourcedistribution.md]: done

File: docs/user-manuals/resourcedistribution.md (hunk 2 above)
Comment (veophi) [reply docs/user-manuals/resourcedistribution.md]: done

Comment (veophi) [reply docs/user-manuals/resourcedistribution.md]: done

**Pull request merged**